integration-tests/spring-zuul-tracing-tests/src/test/java/org/apache/servicecomb/spring/cloud/zuul/tracing/SpringCloudZuulTracingTest.java

```diff
@@ -82,15 +82,18 @@ public void tracesCallsReceivedFromZuulToCalledService() throws InterruptedExcep
   @Test
   public void tracesFailedCallsReceivedByZuul() throws InterruptedException {
     ResponseEntity<String> responseEntity = testRestTemplate.getForEntity("/dummy/rest/oops", String.class);
-
-    assertThat(responseEntity.getStatusCode(), is(INTERNAL_SERVER_ERROR));
-
+    try {
+      int code = responseEntity.getStatusCode().value();
+      assertThat(code, is(590));
+      expectFailing(IllegalArgumentException.class);
```

Comment:
if throw IllegalArgumentException, then failed?

Reply:
expectFailing is very hard to understand how it works